tests/conftest.py

```diff
@@ -46,3 +66,135 @@ def write_clean_ini(path: Path, txt: str) -> None:
         path.write_text(clean_ini)
 
     return write_clean_ini
+
+
+@pytest.fixture(name="matrix_service")
+def matrix_service_fixture() -> MatrixService:
```

Comment:
Note that there is another matrix fixture in a nested `conftest.py`, I guess the nested one takes precedence, but it could be misleading. I don"t have any alternative to propose though :)

Also, cannot we just re-use the `InMemorySimpleMatrixService` instead of this mock ?
The study does not need a full MatrixService, only a `ISimpleMatrixService`